packages/ia-wayback-search/src/example-webcomponent.js

```diff
@@ -0,0 +1,52 @@
+import { LitElement, html, css } from 'lit-element';
+
+export default class HelloComponent extends LitElement {
```

Comment:
Is this example component supposed to be here?

Reply:
Nope. I'll remove it.